contrib/gn_module_validation/backend/gn_module_validation/tasks.py

```diff
@@ -0,0 +1,37 @@
+from celery.schedules import crontab
+from celery.utils.log import get_task_logger
+
+from geonature.core.gn_commons.models import TValidations
+from geonature.utils.celery import celery_app
+from geonature.utils.config import config
+
+logger = get_task_logger(__name__)
+
+
+@celery_app.on_after_finalize.connect
+def setup_periodic_tasks(sender, **kwargs):
+    is_enabled = config["VALIDATION"]["AUTO_VALIDATION_ENABLED"]
+    ct = config["VALIDATION"]["AUTO_VALIDATION_CRONTAB"]
+    if ct and is_enabled:
+        minute, hour, day_of_month, month_of_year, day_of_week = ct.split(" ")
+        sender.add_periodic_task(
+            crontab(
+                minute=minute,
+                hour=hour,
+                day_of_week=day_of_week,
+                day_of_month=day_of_month,
+                month_of_year=month_of_year,
+            ),
+            set_auto_validation.s(),
+            name="auto validation",
+        )
+
+
+@celery_app.task(bind=True)
+def set_auto_validation(self):
```

Comment:
It would be beneficial to add error handling in the `set_auto_validation` task to ensure that any failures in the auto-validation process are properly logged and handled. [important]